kyuubi-server/src/main/scala/org/apache/kyuubi/session/KyuubiBatchSessionImpl.scala

```diff
@@ -89,7 +90,8 @@ class KyuubiBatchSessionImpl(
       batchRequest.getClassName,
       normalizedConf,
       batchRequest.getArgs.asScala,
-      recoveryMetadata)
+      recoveryMetadata,
+      batchRequest.isCleanupResourceFile)
```

Comment:
refer the `sessionName` method of KyuubiBatchSessionImpl.

Reply:
`isCleanupResourceFile` is no longer needed here. Removed.